skyvern/forge/sdk/routes/webhooks.py

```diff
@@ -0,0 +1,128 @@
+import structlog
+from fastapi import Depends, HTTPException, status
+
+from skyvern.exceptions import SkyvernHTTPException, TaskNotFound, WorkflowRunNotFound
+from skyvern.forge.sdk.routes.routers import base_router, legacy_base_router
+from skyvern.forge.sdk.schemas.organizations import Organization
+from skyvern.forge.sdk.services import org_auth_service
+from skyvern.schemas.webhooks import (
+    RunWebhookPreviewResponse,
+    RunWebhookReplayRequest,
+    RunWebhookReplayResponse,
+)
+from skyvern.services.webhook_service import (
+    MissingApiKey,
+    MissingWebhookTarget,
+    WebhookReplayError,
+    build_run_preview,
+    replay_run_webhook,
+)
+
+LOG = structlog.get_logger()
+
+
+@legacy_base_router.get(
+    "/internal/runs/{run_id}/test-webhook",
+    tags=["Internal"],
+    response_model=RunWebhookPreviewResponse,
+    summary="Preview webhook replay payload",
+)
+@base_router.get(
+    "/internal/runs/{run_id}/test-webhook",
+    tags=["Internal"],
+    response_model=RunWebhookPreviewResponse,
+    summary="Preview webhook replay payload",
```

Comment:
add `include_in_schema=False` so this endpoint doesn't end up in our official API doc